app/controllers/publishers/vacancies/job_applications/self_disclosure_controller.rb

```diff
@@ -0,0 +1,49 @@
+class Publishers::Vacancies::JobApplications::SelfDisclosureController < Publishers::Vacancies::JobApplications::BaseController
+  before_action :not_enabled
+
+  helper_method :job_application
```

Comment:
I've been trying to avoid this pattern in favour of just setting controller variables and passing them to the view